db.js

```diff
@@ -301,25 +320,28 @@ exports.init = function (sbot, config) {
       indexName = 'base'
     }
 
-    onIndexesStateLoaded(() => {
-      log.onDrain(() => {
-        const index = indexes[indexName]
-        if (!index) return cb('Unknown index:' + indexName)
-
-        status.updateLog()
-
-        if (index.offset.value === log.since.value) {
-          status.updateIndex(indexName, index.offset.value)
-          cb()
-        } else {
-          const remove = index.offset(() => {
-            if (index.offset.value === log.since.value) {
-              remove()
-              status.updateIndex(indexName, index.offset.value)
-              cb()
-            }
-          })
-        }
+    // setTimeout to make sure extra indexes from secret-stack are also included
+    setTimeout(() => {
```

Comment:
This new setTimeout was important to fix the NaN offset issue. Some onDrain's out there, such as the newly added one in EBT compat, would cause the trigger of updateIndexes() *before* all the extra level plugins had the chance to get registered, which means that the `Math.min` would operate on `undefined` values, and that would result in `NaN` and then it would be a log.stream `gt: NaN`. So we need to delay this just a bit.

Reply:
Nothing new in the body, just a setTimeout and a re-indent. Makes the diff look scary